main.less

```diff
@@ -30,16 +25,24 @@
     border: 1px solid #deca7e;
 }
 
-.errors {
-    width: 25px;
-    cursor: pointer;
-}
+.hint-marker-positioning {
+  left: auto !important; /* CodeMirror will set this value, but we don't want it set! */
+  right: 1em;
+  background-color: transparent;
 
-.gutterCursor {
-    cursor: pointer;
+  margin: 0px;
+  padding: 0px;
+  cursor: pointer;
+
+  width: 1.4em;
+  height: 1.4em;
+  margin-top: -1.3em;
+
+  background-size: 100% 100%;
+  z-index: 999;
 }
 
-.CodeMirror .CodeMirror-linenumber {
-    padding-right: 0px;
-    padding-left: 0px;
+.hint-marker-error {
+  background-image: url("./explanationpt.png");
```

Comment:
This image was included in thimble itself, so I assumed it was free for me to use